CHANGELOG.md

```diff
@@ -1,6 +1,98 @@
+* **New Resource:** `vcd_provider_vdc` to manage provider VDCs ([#1073](https://github.com/vmware/terraform-provider-vcd/pull/1073))
+* **New Data Source:** `vcd_resource_pool` to read vCenter Resource Pools ([#1073](https://github.com/vmware/terraform-provider-vcd/pull/1073))
+* **New Data Source:** `vcd_network_pool` to read Network Pools ([#1073](https://github.com/vmware/terraform-provider-vcd/pull/1073))
+* **New Resource:** `vcd_rde_interface_behavior` to manage RDE Interface Behaviors, which can be invoked by RDEs and
+  overridden by RDE Types ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Data Source:** `vcd_rde_interface_behavior` to read RDE Interface Behaviors, so they can be used
+  in RDE Type Behavior overrides ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Resource:** `vcd_rde_type_behavior` to manage Behaviors in RDE Types, which can override those defined
+  in RDE Interfaces ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Data Source:** `vcd_rde_type_behavior` to read RDE Type Behaviors ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Resource:** `vcd_rde_type_behavior_acl` to manage the access to Behaviors in RDE Types and RDE Interfaces ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Data Source:** `vcd_rde_type_behavior_acl` to read Access Levels from Behaviors of RDE Types and RDE Interfaces ([#1074](https://github.com/vmware/terraform-provider-vcd/pull/1074))
+* **New Resource:** `vcd_nsxt_edgegateway_static_route` to manage NSX-T Edge Gateway Static Routes
+  on VCD 10.4.0+ ([#1075](https://github.com/vmware/terraform-provider-vcd/pull/1075))
+* **New Data Source:** `vcd_nsxt_edgegateway_static_route` to read NSX-T Edge Gateway Static Routes
+  on VCD 10.4.0+ ([#1075](https://github.com/vmware/terraform-provider-vcd/pull/1075))
+* **New Resource:** `vcd_nsxt_distributed_firewall_rule` to manage NSX-T Distributed Firewall one by
+  one. Rules will *not be created in parallel* because the API provides no direct endpoint to create
+  a single rule and this functionality uses a custom-made function that abstracts the "update all"
+  endpoint ([#1076](https://github.com/vmware/terraform-provider-vcd/pull/1076))
+* **New Data Source:** `vcd_nsxt_distributed_firewall_rule` to read NSX-T Distributed Firewall one
+  by one ([#1076](https://github.com/vmware/terraform-provider-vcd/pull/1076))
+* **New Resource:** `vcd_cloned_vapp` to create a vApp from either a vApp template or another vApp ([#1081](https://github.com/vmware/terraform-provider-vcd/pull/1081))
+
+### IMPROVEMENTS
+* Add `ignore_metadata_changes` argument to the Provider configuration to be able to specify metadata entries that should **not**
+  be managed by Terraform when using `metadata_entry` configuration blocks ([#1057](https://github.com/vmware/terraform-provider-vcd/pull/1057))
```

Comment:
Just adding a reminder that we may need to add
https://github.com/vmware/terraform-provider-vcd/pull/1089/files 
```
([#1057](https://github.com/vmware/terraform-provider-vcd/pull/1057), [#1089](https://github.com/vmware/terraform-provider-vcd/pull/1089))
```